### DISCUSSION TITLE
Additional Search Filtering (Titles, Headings, etc.)

### DISCUSSION THREAD

**Discussion by mweth** [outline/outline#4475] [Ideas]
I do not quite understand how the search function prioritizes results.  Sometimes, I will have a doc or collection that is titled XYZ, but the search will not pull up the top level exact name match first.

It would be helpful to be able to search in:title or simply having a filter to search in headings and/or document title.

**Comment by tommoor** [comment]
We're tracking this one: 
https://github.com/outline/outline/issues/4192

In general in search results if the term appears in the title it is returned higher in the results, but this could be overridden if the same term appeared many times in the body of another document for example.
